DSCResources/MSFT_xWebVirtualDirectory/MSFT_xWebVirtualDirectory.psm1

```diff
@@ -9,6 +9,7 @@ function Get-TargetResource
         $Website,
 
         [parameter(Mandatory = $true)]
```

Comment:
If we're changing this to accept nulls, does it really need to be mandatory anymore?